mailpoet/tests/integration/EmailEditor/Engine/EmailEditorTest.php

```diff
@@ -42,6 +44,29 @@ public function testItRestrictsAllowedBlockTypes() {
     expect(count((array)$blockTypes))->equals(4);
   }
 
+  public function testItWorksWithEmptyEditorContext() {
+    $settings = [
+      'test' => 1,
+    ];
+    $blockTypes = [
+      'core/paragraph',
+    ];
+    $wp = $this->diContainer->get(Functions::class);
+    $wp->removeAllFilters('block_editor_settings_all');
+    $wp->removeAllFilters('allowed_block_types_all');
+    $this->emailEditor->initialize();
+    // Test without the context argument
+    $filteredSettings = $wp->applyFilters('block_editor_settings_all', $settings);
+    $filteredBlockTypes = $wp->applyFilters('allowed_block_types_all', $blockTypes);
+    expect($filteredSettings)->equals($settings);
+    expect($filteredBlockTypes)->equals($blockTypes);
+    // Test without the context argument as null
```

Comment:
If I understand correctly, this comment should be `Test with the context argument as null`. Is it right?